test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpImplicitExpressionTest/ParsesNullConditionalOperatorImplicitExpression_Bracket9.stree.txt

```diff
@@ -1,5 +1,8 @@
-Expression block - Gen<Expr> - 4 - (0:0,0)
-    Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-        SyntaxKind.Transition;[@];
-    Code span - Gen<Expr> - [val] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (1:0,1) - Tokens:1
-        SyntaxKind.Identifier;[val];
+CSharpCodeBlock - [0..4)::4 - [@val]
```

Comment:
ditto. Given that this and the other one exist. Just verifying, are these test outputs correct? They seem too simple